### PR TITLE
Docs style guide

### PR DESCRIPTION
**Pull request by andriitserkovnyi**
## PR Description

Add guidance for style

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/scos/user/intro-to-spryker/contribute-to-the-documentation/spryker-documentation-style-guide/spryker-documentation-style-guide.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Use the following references in this order:
+
+1. Spryker documentation style guide
+2. [Google developer documentation style guide](https://developers.google.com/style)
+3. [Microsoft Writing Style Guide](https://learn.microsoft.com/en-us/style-guide/welcome/)
+
+## Write in a way that makes sense
+
+Spryker is a rapidly evolving product, as well as its documentation. If it makes sense to not follow the guides, go for it. Consider if your approach is better only in your  particular case or for the entire documentation. Edit existing and add new guidance to this style guide.
```
Comment (lenadoc) [inline docs/scos/user/intro-to-spryker/contribute-to-the-documentation/spryker-documentation-style-guide/spryker-documentation-style-guide.md:22]:
```suggestion
Spryker is a rapidly evolving product, as well as its documentation. If it makes sense to not follow the guides, go for it. Consider if your approach is better only in your particular case or for the entire documentation. Edit existing and add new guidance to this style guide.
```

**Review by lenadoc — CHANGES_REQUESTED** [review]

File: docs/scos/user/intro-to-spryker/contribute-to-the-documentation/spryker-documentation-style-guide/examples.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+template: concept-topic-template
+---
+
+In general, follow [Google's guidance for examples](https://developers.google.com/style/format-examples?hl=en).
+
+Additional guidance for introducing short and long examples:
+
+<div class="width-100">
+
+| Guidance | Recommended | Not recommended |
```
Comment (VadymSachenko) [inline docs/scos/user/intro-to-spryker/contribute-to-the-documentation/spryker-documentation-style-guide/examples.md:14]: We use upper case for table headers
```suggestion
| GUIDENCE | RECOMMENDED | NOT RECOMMENDED |
```

File: docs/scos/user/intro-to-spryker/contribute-to-the-documentation/spryker-documentation-style-guide/spryker-documentation-style-guide.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+When writing and reviewing documentation at Spryker, we are mostly relying on external style guides.
+
+The goal of our own style guide is to provide guidance for the cases that are specific to Spryker or those not covered by the external style guides.
+
+## Reference hierarchy
+
+Use the following references in this order:
+
+1. Spryker documentation style guide
```
Comment (VadymSachenko) [inline docs/scos/user/intro-to-spryker/contribute-to-the-documentation/spryker-documentation-style-guide/spryker-documentation-style-guide.md:16]: Do you know where I can read this one?

Comment (andriitserkovnyi) [reply docs/scos/user/intro-to-spryker/contribute-to-the-documentation/spryker-documentation-style-guide/spryker-documentation-style-guide.md:16]: Here :)

**Review by VadymSachenko — APPROVED** [review]

**Review by AlexSlawinski — APPROVED** [review]

**Review by AlexSlawinski — APPROVED** [review]

**Review by lenadoc — APPROVED** [review]

**Pull request merged**